exercises/pig-latin/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "pig-latin"
+version = "0.1.0"
```

Comment:
since this exercise complies with https://github.com/exercism/problem-specifications/blob/master/exercises/pig-latin/canonical-data.json which is version 1.0.0, could you change this version here to 1.0.0?

It is not your fault that you did not know to do this, since this track has failed to document it (https://github.com/exercism/rust/issues/278#issuecomment-299741390)